tests/software_tests/can/packet/test_consecutive_frame.py

```
@@ -0,0 +1,590 @@
+                                            target_address=target_address,
+                                            address_extension=address_extension)
+        self.mock_can_addressing_information.encode_ai_data_bytes.assert_called_once_with(
+            addressing_format=addressing_format,
+            target_address=target_address,
+            address_extension=address_extension)
+        self.mock_can_dlc_handler.decode_dlc.assert_called_once_with(dlc)
+
+    # extract_consecutive_frame_payload
+
+    @pytest.mark.parametrize("addressing_format, raw_frame_data, ai_bytes_number", [
+        (Mock(), [0x2F] + list(range(63)), 0),
```

Ruff (0.12.2)

Consider `[47, *list(range(63))]` instead of concatenation

Replace with `[47, *list(range(63))]`

(RUF005)